apps/remote_control/lib/lexical/remote_control/search/fuzzy/scorer.ex

```diff
@@ -0,0 +1,189 @@
+defmodule Lexical.RemoteControl.Search.Fuzzy.Scorer do
```

Comment:
This whole module is super interesting! It'll also be interesting to see how it performs in the real world and how we can tune and adapt it over time.

Reply:
It performs pretty well right now. I can search over all of the module references (~50,000) in around 30ms.

Reply:
it also builds and updates quickly.

Reply:
Sorry I wasn't very clear. By performance I meant the accuracy of results, not "cpu cycles" performance. Glad to know it's fast though!